docs/Release_Guide/metcalcpy_bugfix.rst

```diff
@@ -11,5 +11,8 @@ Create a new vX.Y.Z bugfix release from the main_vX.Y branch.
 .. include:: release_steps/metcalcpy/update_version_bugfix.rst
 .. include:: release_steps/update_release_notes_bugfix.rst
 .. include:: release_steps/metcalcpy/update_docs_bugfix.rst
+.. include:: release_steps/merge_release_issue.rst
 .. include:: release_steps/create_release_on_github.rst
+.. include:: release_steps/metcalcpy/create_release_extra.rst
+.. include:: release_steps/metplus/update_dtc_website.rst
 .. include:: release_steps/finalize_release_on_github_official.rst
```

Comment:
```suggestion
.. include:: release_steps/finalize_release_on_github_bugfix.rst
```
Could you please update this to be the bugfix version instead of the official version?